datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
+            inspector = inspect(conn)
+            schema = inspector.get_schema_names()  # returns a list
+            views = inspector.get_view_names(
+                schema[0])  # returns a list
+
+            for view in views:
+                definition = inspector.get_view_definition(view, schema[0])
+                yield (view, definition)
+                # print(view, definition)
+
+    def get_lineage_for_column(self, view_name: str, view_sql: str) -> Iterable[Tuple[str, Any]]:
+
+        cols = parse_one(view_sql).named_selects
+        for col_name in cols:
+            yield (view_name, lineage(col_name.lower(), view_sql.lower()))
+
+    def parse_column_name(self, column_name: str):
+        parsed_name = column_name.split(".")
+        return parsed_name[1]
+
+    def get_column_view_lineage_elements(self) -> Iterable[Tuple[Field, List[UpstreamField]]]:
```

Comment:
for functions that return rather nested types like this, i recommend writing a doc string and include a description of what the return type actually mean, sometimes even an pseudo-code example.